src/App.js

```diff
@@ -130,6 +131,7 @@ const App = ({ locale, location, dependencies: { appSessionRef, sessionManager }
                 component={ReportsPartialsCampaigns}
               />
               <PrivateRoute path={['/billing/invoices']} exact component={InvoicesList} />
+              <PrivateRoute path={'/contact-policy'} exact component={ContactPolicy} />
```

Comment:
Here the final url will be `https://app.fromdoppler.com/contact-policy`  wouldn`t be better to keep Control panel urls styles? Something like this:
```suggestion
              <PrivateRoute path={'ControlPanel/CampaignPreferences/contact-policy'} exact component={ContactPolicy} />
```

Reply:
Sounds good to me, let's use the defined routes for English and Spanish.
EN: ControlPanel/CampaignsPreferences/ContactPolicy
ES: PaneldeControl/PreferenciasdeCampana/PoliticadeContacto